packages/functional-tests/tests/syncV3/syncV3ResetPassword.spec.ts

```diff
@@ -5,94 +5,56 @@
-    email = login.createEmail();
-    await target.auth.signUp(email, PASSWORD, {
-      lang: 'en',
-      preVerified: 'true',
-    });
-    await login.clearCache();
-    await page.goto(
-      `${
-        target.contentServerUrl
-      }/reset_password?context=fx_desktop_v3&service=sync/?${queryParam.toString()}`
-    );
-    await resetPassword.fillOutResetPassword(email);
-
-    const link = await target.email.waitForEmail(
-      email,
-      EmailType.recovery,
-      EmailHeader.link
-    );
-    await page.goto(link, { waitUntil: 'load' });
-    await resetPassword.resetNewPassword('Newpassword@');
-    expect(await login.loginHeader()).toBe(true);
-  });
-
-  test('reset password, verify same browser, password validation', async ({
+  test('reset pw, test pw validation, verify same browser', async ({
+    credentials,
     target,
   }) => {
     const { page, login, resetPassword } = await newPagesForSync(target);
-    const query = {
-      forceExperiment: 'passwordStrength',
-      forceExperimentGroup: 'designF',
-    };
-    email = login.createEmail();
-    await target.auth.signUp(email, PASSWORD, {
-      lang: 'en',
-      preVerified: 'true',
-    });
-    await login.clearCache();
     await page.goto(
       `${target.contentServerUrl}/reset_password?context=fx_desktop_v3&service=sync`
     );
-    await resetPassword.fillOutResetPassword(email);
+    await resetPassword.fillOutResetPassword(credentials.email);
 
     const link = await target.email.waitForEmail(
-      email,
+      credentials.email,
       EmailType.recovery,
       EmailHeader.link
     );
-    await resetPassword.addQueryParamsToLink(link, query);
     await page.goto(link, { waitUntil: 'load' });
 
-    //Enter a short password
+    // Enter a short password
     await resetPassword.resetNewPassword('pass');
 
-    //Verify the error
+    // Verify the error
     expect(await login.minLengthFailError()).toBe(true);
     expect(await login.notEmailUnmetError()).toBe(true);
     expect(await login.notCommonPasswordUnmetError()).toBe(true);
 
-    //Enter a common password
+    // Enter a common password
     await resetPassword.resetNewPassword('password');
 
-    //Verify the error
+    // Verify the error
     expect(await login.minLengthSuccess()).toBe(true);
     expect(await login.notEmailSuccess()).toBe(true);
     expect(await login.notCommonPasswordFailError()).toBe(true);
 
-    //Enter the email as password
-    await resetPassword.resetNewPassword(email);
+    // Enter the email as password
+    await resetPassword.resetNewPassword(credentials.email);
 
-    //Verify the error
+    // Verify the error
     expect(await login.minLengthSuccess()).toBe(true);
     expect(await login.notEmailFailError()).toBe(true);
     expect(await login.notCommonPasswordUnmetError()).toBe(true);
+
+    await resetPassword.resetNewPassword('Newpassword@');
+    expect(await resetPassword.completeResetPasswordHeader()).toBe(true);
```

Comment:
In a recent PR I merged, I had removed the `expect...toBe(true)` when using `await`. I feel like we should verify that soon so we can stop adding those or fix places where we might need to add it 😬